README.md

```diff
@@ -16,16 +15,173 @@ To set up the project, download these tools:
+APP_CONFIG_FILE_NAME=contrib/minimal-config.yaml \
+APP_NAMESPACE=kyma-system go run cmd/broker/main.go
+```
+
+```bash
+APP_KUBECONFIG_PATH=/Users/$User/.kube/config \
+APP_DOCUMENTATION_ENABLED=false \
+APP_TMP_DIR=/tmp APP_NAMESPACE=default \
+APP_SERVICE_NAME=helm-broker \
+APP_CONFIG_FILE_NAME=contrib/minimal-config.yaml \
+APP_CLUSTER_SERVICE_BROKER_NAME=helm-broker \
+APP_DEVELOP_MODE=true \
+go run cmd/controller/main.go -metrics-addr ":8081"
+```
+
+### Use environment variables
+
+Use the following environment variables to configure the application:
+
+#### Broker
+
+| Name | Required | Default | Description |
+|-----|---------|--------|------------|
+| **APP_PORT** | No | `8080` | The port on which the HTTP server listens. |
+| **APP_KUBECONFIG_PATH** | No |  | The path to the `kubeconfig` file that you need to run an application outside of the cluster. |
+| **APP_CONFIG_FILE_NAME** | No | | Specifies the path to a config YAML file.|
+| **APP_HELM_TILLER_TLS_ENABLED** | No | true | Specifies the tls configuration with the Tiller. If `true` the TLS communication with Tiller is required. |
+| **APP_HELM_TILLER_HOST** | No | | Specifies the host address of the Tiller release server. |
+| **APP_HELM_TILLER_INSECURE** | No | false | Controls whether the broker verifies the Tiller's certificate. |
+| **APP_HELM_TILLER_KEY** | No | | A path to PEM-encoded private key file. |
+| **APP_HELM_TILLER_CRT** | No | | A path to PEM-encoded certificate file. |
+
+#### Controller
+
+| Name | Required | Default | Description |
+|-----|---------|--------|------------|
+| **APP_CONFIG_FILE_NAME** | No | | Specifies the path to a config YAML file.|
+| **APP_TMP_DIR** | Yes | | Path to a temporary directory, used for unpacking addons archives or cloning Git repositories. |
+| **APP_KUBECONFIG_PATH** | No |  | The path to the `kubeconfig` file that you need to run an application outside of the cluster. |
+| **APP_NAMESPACE** | Yes | | The namespace, where Helm-Broker is installed. |
+| **APP_SERVICE_NAME** | Yes | | The name of the Kubernetes Service which is exposing the broker. |
+| **APP_CLUSTER_SERVICE_BROKER_NAME** | Yes | | The name of the ClusterServiceBroker resource which registers the Helm-Broker into the Service-Catalog. |
+| **APP_DEVELOP_MODE** | No | false | If `true` insecure HTTP based repositories are allowed. |
+| **APP_DOCUMENTATION_ENABLED** | No | false | Indicates if Helm-Broker should upload addons documentation to the [Headless CMS](https://kyma-project.io/docs/components/headless-cms/). |
+
+## Development
+
+Before each commit, use the `before-commit.sh` script, which tests your changes.
 
+Integration tests checks if all part of Helm-Broker works together. It starts `kube-apiserver` with `etcd`, setups the controller and the broker. Before you run it, please install [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder). You can run integration tests using the follwoing command:
```

Comment:
Explain how to develop a change, build image push and update chart and install upgrade a chart.